### PR TITLE
Images for streaming get involved pages

### PR DESCRIPTION
**Pull request by niallrobinson**

### CODE REVIEW

**Comment by niallrobinson** [comment]
not ready for pull yet

File: get-involved/index.md [hunk 1]
```diff
@@ -6,13 +6,26 @@ author: Niall Robinson
...
-
-###Getting set-up
-If you are working closely with the Lab, you'll need some information to get set up.
-
-* [How to contribute to the blog](./how-to-blog)
+<div class="container-fluid">
+	<div class="row">
+		<div class="col-md-3">
+			<a href="coder/" class="thumbnail">
+				<img src="/images/coder.svg"/, alt="coder">
```
Comment (jacobtomlinson) [inline get-involved/index.md]: @niallrobinson There is a comma in here for some reason

Comment (niallrobinson) [inline get-involved/index.md]: see above comment - i'm just claiming the PR.

**Comment by niallrobinson** [comment]
@jacobtomlinson @simonstanley First cut is almost done. I'm just having a problem making the bounding boxes appear the full heigh of their row when the <p> text is at different stages of wrapping than its partner. Hope that makes sense.

**Comment by niallrobinson** [comment]
@simonstanley ready for a merge if it passes Travis

**Pull request merged**